### PR TITLE
Send grafana through nginx

### PR DESCRIPTION
**Pull request by Wmbat**

### CODE REVIEW

File: tasks/firewall.yml [hunk 1]
```diff
@@ -12,3 +12,8 @@
   ufw:
```
Comment (charlesfleche) [inline tasks/firewall.yml:12]: The 3000 port should be closed

File: files/nginx/grafana.conf [hunk 2]
```diff
@@ -0,0 +1,15 @@
+server {
+    listen      80;
+    listen      [::]:80 ipv6only=on;
```
Comment (charlesfleche) [inline files/nginx/grafana.conf:3]: What does ipv6only=on do ?

Comment (Wmbat) [reply files/nginx/grafana.conf:3]: it tell nginx to only use this line if it's receiving an ipv6 address

Comment (charlesfleche) [reply files/nginx/grafana.conf:3]: I don't think this is needed anymore in recent versions of NGinx.
https://serverfault.com/questions/842492/nginx-duplicate-listen-options-for-80-error/842515

**Pull request merged**